### PR TITLE
Clean up PR

### PR DESCRIPTION
**Pull request by henrykironde**

### CODE REVIEW

File: docs/pyretriever.rst [hunk 1]
```diff
@@ -77,25 +72,27 @@ There are a lot of different attributes provided in the Scripts class. Some nota
...
-Downloading for the first time the functions will take sometime depending on the internet connection.
-In case of poor internet connection or other problems an error is raised.
+Downloading recipes for all datasets can take a while depending on the internet connection.
 
 Download Datasets
 =================
 
-To download datasets the ``download`` function can be used.
-It has the following function definition.
+To directly download datasets without cleaning them use the download function
```
Comment (ShivamNegi) [inline docs/pyretriever.rst]: The \``download\`` function

File: docs/pyretriever.rst [hunk 2]
```diff
@@ -114,12 +111,12 @@ Output:
   => Downloading iris
 
   Downloading bezdekIris.data...
-  100%  0 seconds Copying bezdekIris.data from /home/user_name/.retriever/raw_data/iris
+  100%  0 seconds Copying bezdekIris.data
+
 
+This downloads the dataset in your current working directory.
+You can control where the dataset is downloaded using the path parameter.
```
Comment (ShivamNegi) [inline docs/pyretriever.rst]: The \``path`` parameter.

File: docs/pyretriever.rst [hunk 3]
```diff
@@ -147,7 +144,7 @@ Retriever supports scripts installation into 7 major formats or engines as we ca
   xml
 
 
-The function definition of the functions is as follows:
+here are separate functions for installing into each of the 7 backends:
```
Comment (ShivamNegi) [inline docs/pyretriever.rst]: 'T' is missed.

File: docs/pyretriever.rst [hunk 4]
```diff
@@ -134,7 +131,7 @@ There are in all 4 default parameters.
 Install Datasets
 ================
 
-Retriever supports scripts installation into 7 major formats or engines as we call them.
+Retriever supports the installation of datasets into 7 major databases and file formats.
```
Comment (ShivamNegi) [inline docs/pyretriever.rst]: `Retriever supports installation of datasets into 7 major databases and file formats.` Extra `the`.

**Review by ShivamNegi — CHANGES_REQUESTED** [review]

**Comment by henrykironde** [comment]
@ShivamNegi, done

**Comment by ShivamNegi** [comment]
LGTM.

**Comment by henrykironde** [comment]
Thanks @ShivamNegi

**Pull request merged**